common/src/main/java/io/pravega/common/util/ServiceBuilderConfig.java

```diff
@@ -7,9 +7,8 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.segmentstore.server.store;
+package io.pravega.common.util;
```

Comment:
This is not a utility class. I'm not sure this needs to be in common either.
Please find a different location for it.